### DISCUSSION TITLE
Compiling richardg867's branch of 86Box

### DISCUSSION THREAD

**Discussion by nt4boy** [86Box/86Box#4886] [Q&A] [answered]
[86BoxCompErrors.txt](https://github.com/user-attachments/files/17267233/86BoxCompErrors.txt)
https://github.com/richardg867/86Box
I would like to try out the PCI passthrough for a non GPU device, and as I understand it, richardg's branch should do that. Cmake completes without error, However, make shuts down with the attached errors relating to vfio.c.o.
I've an idea that this likely isn't the correct place or way to report and ask for assistance resolving this, but I'd be grateful to be pointed to where I can do this properly. Thank you.

**Comment by jriwanek** [comment, marked as answer]
vfio is now part of master.

**Comment by nt4boy** [comment]
Thank you for letting me know.
Will test.

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________
